packages/babel-plugin-transform-react-jsx/test/fixtures/react/handle-spread-with-proto-babel-7/input.js

```diff
@@ -0,0 +1,7 @@
+var __proto__ = null;
+
+<p {...{__proto__: null}}>text</p>;
+
+<div {...{"__proto__": null}}>{contents}</div>;
+
+<img alt="" {...{__proto__}} />;
```

Comment:
This should behave like the computed fields, could you move it to `packages/babel-plugin-transform-react-jsx/test/fixtures/react/flattens-spread/input.js`?